dependencies.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
We already have `_build_dependencies.sh`. Please integrate your changes into that file.